firehose_replicator/firehose_replicator.py

```diff
@@ -0,0 +1,53 @@
+import base64
+import boto3
+import json
+import os
+from datetime import datetime
+import uuid
+import gzip
+
+def lambda_handler(event, context):
+    output = []
+    records_for_s3 = []
+    stream_name = event['deliveryStreamArn'].split('/')[1]
+    target_bucket = os.environ.get('ECS_BUCKET_NAME')
```

Comment:
Can we have this be more generic so when its deployed to ECS and pushes to quarantine it will still make sense to a reader? Something like `MIRRORED_BUCKET_NAME`.

Reply:
yup - until now I had assumed we were going one-way